app/models/AppResult.java

```diff
@@ -158,6 +158,9 @@ public static String getSearchFields() {
   @Column(nullable = true)
   public long totalDelay;
 
+  @Column(length = 255, nullable = true)
+  public String parents;
```

Comment:
Parent is a comma separated list of parents right. What is the consequence if length grows over 255? Do you drop it or truncate parent names?